seismicpro/src/refractor_velocity/utils.py

```diff
@@ -36,3 +39,84 @@ def postprocess_params(params):
+
+    Parameters
+    ----------
+    rv : RefractorVelocity
+        RefractorVelocity instance.
+
+    Returns
+    -------
+    df : pandas.DataFrame
+        DataFrame with the coordinates and parameters of a RefractorVelocity.
+    """
+    columns = ['name_x', 'name_y', 'coord_x', 'coord_y'] + list(rv.params.keys()) + ["max_offset"]
+    data = [*rv.coords.names] + [*rv.coords.coords] + list(rv.params.values()) + [rv.max_offset]
+    return pd.DataFrame.from_dict({col: [data] for col, data in zip(columns, data)})
+
+def dump_rv(df_list, path, encoding, min_col_size):
+    """Dump list of DataFrame to a file.
+
+    Each DataFrame in the list should have next structure:
+
+     - Columns contain the Coordinates parameters names (name_x, name_y, coord_x, coord_y) and the RefractorVelocity
+     parameters names ("t0", "x1"..."x{n-1}", "v1"..."v{n}", "max_offset").
+     - First row contains the coords names, coords values, and parameters values.
+
+    DataFrame example :
+         name_x      name_y     coord_x     coord_y        t0        x1        v1        v2 max_offset
+    0   SourceX     SourceY     1111100     2222220     50.00   1000.00   1500.00   2000.00    2000.00
+
+    Parameters
+    ----------
+    df_list : iterable of :class:`~pandas.DataFrame`
+        Each DataFrame in the passed list should contain coordinates and parameters of a :class:`~RefractorVelocity`.
+    path : str
+        Path to the created file.
+    encoding : str, optional, defaults to "UTF-8"
+        File encoding.
+    min_col_size : int
+        Minimum size of each columns in the resulting file.
+
+    Returns
+    -------
+    None
+    """
+    with open(path, 'w', encoding=encoding) as f:
+        pd.concat(df_list).to_string(buf=f, col_space=min_col_size, float_format="%.2f", index=False)
+
+def load_rv(path, encoding):
+    """Load the coordinates and parameters of RefractorVelocity from a file.
+
+    Parameters
+    ----------
+    path : str
+        Path to the file.
+    encoding : str, optional, defaults to "UTF-8"
+        File encoding.
+
+    Returns
+    -------
+    coords_list : list of :class:`~utils.Coordinates`
+        List of Coordinates instances loaded from a file.
+    params_list : list of dict
+        List of parameters of :class:`~RefractorVelocity`.
+    max_offset_list : list of float
+        List of max offsets.
+    """
+    df = pd.read_csv(path, sep=r'\s+', encoding=encoding)
+    n_refractors = (len(df.columns) - 5) // 2
+    coords_list, params_list, max_offset_list = [], [], []
+    for row in df.to_numpy():
+        if not all([isinstance(names, str) for names in row[:2]] + [isinstance(val, (int, float)) for val in row[2:]]):
+            raise ValueError(f"Wrong parameter type in the row {row} to create a correct RefractorVelocity instance.")
+        if np.any(np.isnan(row[-1])):
+            raise ValueError(f"Unsufficient parameters in the row {row} to create a correct RefractorVelocity "
```

Comment:
anyway, do you really need this check? under what conditions can you get `np.nan`?
Maybe change it to `np.isfinite`?

```suggestion
            raise ValueError(f"Found unsufficient parameters in the row {row}"
```

Reply:
The last item in a row will `np.nan` if a row has less values than expected.
The number of expected parameters is defined by first row with names of parameters.
If any row contains more values that expected, the pandas backend will raise ParseError.